cmd/kubeadm/.import-restrictions

```diff
@@ -121,7 +120,6 @@
 				"github.com/docker/go-connections/sockets",
 				"github.com/docker/go-connections/tlsconfig",
 				"github.com/docker/go-units",
-				"github.com/docker/libnetwork/ipvs",
```

Comment:
this is still pulled through the following dependency:

`k8s.io/kubernetes/pkg/util/ipvs`-> `k8s.io/kubernetes/pkg/proxy/ipvs` -> `k8s.io/kubernetes/cmd/kubeadm/app/preflight`

TL;DR - we either need to copy `CanUseIPVSProxier` that is under `pkg/proxy/ipvs` or find a new home for it

Reply:
I'll try to ping some kube-proxy folks about `k8s.io/kubernetes/pkg/proxy/ipvs`. This code needs to be shared and maintained by kube-proxy folks, so that it does not become outdated quickly. We should copy it as a last resort only.

Reply:
@rosti - agreed

Reply:
cc @kubernetes/sig-network-misc

Reply:
Looking at the code, I think, that `GetKernelVersionAndIPVSMods` needs to be moved to `k8s.io/utils/net` and `CanUseIPVSProxier` to `k8s.io/kube-proxy/ipvs`.

Reply:
We're still calling CanUseIPVSProxier in this PR, wondering why the import restriction here passes now?

Reply:
nvm, it's failing :P

Reply:
responded that we should remove CanUseIPVSProxier on slack:
https://kubernetes.slack.com/archives/C13J86Z63/p1562973133411800

> k8s.io/kube-proxy/ipvs

seems like a good location, but i'm starting to think that we should call kube-proxy to tell us if the system is capable.
same goes for conversion and validations, @rosti .